custom_components/tahoma/sensor.py

```diff
@@ -1,21 +1,19 @@
 """Support for TaHoma sensors."""
 from datetime import timedelta
 import logging
-from typing import List, Optional
+from typing import Optional
 
 from homeassistant.components.sensor import DOMAIN as SENSOR
 from homeassistant.const import (
-    CONCENTRATION_PARTS_PER_MILLION,
     DEVICE_CLASS_HUMIDITY,
     DEVICE_CLASS_ILLUMINANCE,
     DEVICE_CLASS_POWER,
     DEVICE_CLASS_TEMPERATURE,
     ENERGY_WATT_HOUR,
-    POWER_WATT,
     TEMP_CELSIUS,
     TEMP_FAHRENHEIT,
     TEMP_KELVIN,
-    UNIT_PERCENTAGE,
+    VOLUME_CUBIC_METERS,
```

Comment:
Why remove percentage? that's the `CORE_RELATIVE_HUMIDITY_STATE` unit.

Reply:
I was not able to find any device returning such MeasuredValueType. Even within Jeedom, there is no such case: https://github.com/redbug26/jeedom-tahoma/blob/6e53512c562ad980f83708313d602db1c05d844e/core/class/tahoma.class.php#L395

Reply:
@tetienne we also miss `CONCENTRATION_PARTS_PER_MILLION` for CO and CO2 sensors.  And possibly also `POWER_WATT` or `ENERGY_KILO_WATT_HOUR`.